### PR TITLE
Additional Clarification on "Why ICMP" (Addresses and closes #33)

### LINKED ISSUE #33: Add some more clarification on why ICMP is apt for delivery of environmental metrics
I believe, should we, in the intro section add some more clarification on why ICMP is apt for delivering the environmental metrics over ICMP (on-demand)?

### PR DESCRIPTION
**Pull request by jmparikh**
Added extra clarification text for Why ICMP was chosen as a medium for transporting environmental sustainability information and close #33.

### CODE REVIEW

**Review by mwelzl — APPROVED** [review]
This is ok with me (and thanks!);  I'm not too fond of "it works very nicely"... I would remove "very nicely", but this is non-blocking to me  :-)

File: draft-pignataro-eimpact-icmp.xml [hunk 1]
```diff
@@ -114,7 +114,11 @@
         increasing demand for monitoring the sustainability and environmental 
         impact. This document serves that need by defining an ICMP extension 
         object that appends environmental sustainability information to 
-        ICMP messages.
+        ICMP messages. Using ICMP as a medium to deliver environmental 
+        sustainability information is very apt as ICMP is one of the most used
+        protocols for administration and troubleshooting and it works very nicely
```
Comment (cpignata) [inline draft-pignataro-eimpact-icmp.xml:119]: replace "very nicely" with "effectively" or similar.

File: draft-pignataro-eimpact-icmp.xml [hunk 2]
```diff
@@ -114,7 +114,11 @@
...
+        in not only an automated setting, but also in an on-demand setting for
```
Comment (cpignata) [inline draft-pignataro-eimpact-icmp.xml:120]: in not only -> not only in

**Review by cpignata — CHANGES_REQUESTED** [review]

**Comment by jmparikh** [comment]
Very valid point. "effectively" is way better than "very nicaly".
Have changed it.

**Review by cpignata — APPROVED** [review]

**Pull request merged**